revscoring/assets/enwiktionary_idioms.txt

```diff
@@ -381,131 +1147,696 @@
 "flea in one's ear"
 "flick the bean"
 "flight of fancy"
-"flip (one's lid|the bird)"
+"flip one's lid"
+"flip the bird"
 "float someone's boat"
-"flog (a dead horse|a dead pony|the dolphin|the log)"
+"flog a dead horse"
+"flog a dead pony"
+"flog the dolphin"
+"flog the log"
 "flood the zone"
 "floppy infant syndrome"
 "flower of the flock"
 "flutter in the dovecote"
-"fly (by the seat of one's pants|in the face of|in the ointment|off at a tangent|off the shelves|on the wall|out of the traps|the coop|the flag|too close to the sun|under the radar)"
-"fold (one's tent)"
-"follow (in someone's footsteps|the crowd)"
+"fly by the seat of one's pants"
+"fly in the face of"
+"fly in the ointment"
+"fly off at a tangent"
+"fly off the shelves"
+"fly on the wall"
+"fly out of the traps"
+"fly the coop"
+"fly the flag"
+"fly too close to the sun"
+"fly under the radar"
+"fold one's tent"
+"follow in someone's footsteps"
+"follow the crowd"
 "food for thought"
 "footnote in history"
-"for (a change|a fact|a song|a start|all one is worth|all the world|crying out loud|fuck's sake|fudge's sake|good and all|good measure|goodness' sake|goodness' sakes|heaven's sake|mercy's sake|my money|old times' sake|one's life|one's particular|Pete's sake|pity's sake|real and for true|that matter|the ages|the asking|the birds|the heck of it|the hell of it|the love of|the love of me|the most part|the nonce|the time being|two pins|XYZ reasons)"
-"force (of habit|someone's hand)"
+"for a change"
+"for a fact"
+"for a song"
+"for a start"
+"for all one is worth"
+"for all the world"
+"for crying out loud"
+"for fuck's sake"
+"for fudge's sake"
+"for good and all"
+"for good measure"
+"for goodness' sake"
+"for goodness' sakes"
+"for heaven's sake"
+"for mercy's sake"
+"for my money"
+"for old times' sake"
+"for one's life"
+"for one's particular"
+"for Pete's sake"
+"for pity's sake"
+"for real and for true"
+"for that matter"
+"for the ages"
+"for the asking"
+"for the birds"
+"for the heck of it"
+"for the hell of it"
+"for the love of"
+"for the love of me"
+"for the most part"
+"for the nonce"
+"for the time being"
+"for two pins"
+"for XYZ reasons"
+"force of habit"
+"force someone's hand"
 "forgive and forget"
 "forty minutes of hell"
 "fountain of youth"
 "Four Horsemen of the Apocalypse"
-"four (score and seven years ago|sheets to the wind)"
+"four score and seven years ago"
+"four sheets to the wind"
 "fox in the henhouse"
-"fresh (off the boat|out of)"
+"fresh off the boat"
+"fresh out of"
 "fret the gizzard"
-"friend (of Bill W.|with benefits)"
+"friend of Bill W."
```

Comment:
Will this `.` be escaped afterwards?

Reply:
Hello @he7d3r Thanks for pointing out. I'll separate those.